pkg/apiserver/apiserver.go

```diff
@@ -16,13 +16,14 @@ import (
 
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
-	"github.com/cloud-ark/kubediscovery/pkg/discovery"
+	"github.com/danielpygo/kubediscovery/pkg/discovery"
```

Comment:
Can you research into how to ensure we can use reusable import paths?

Reply:
Ok I asked a question on stackoverflow